app/models/BotRequest.js

```diff
@@ -57,7 +57,7 @@ botRequestSchema.statics.log = function (req, botType, msgType, msg) {
     data.user_id = req.user._id;
   }
   if (req.campaign) {
-    data.campaign_id = req.campaign._id;
+    data.campaign_id = req.campaign.id;
```

Comment:
There used to be a `Campaign` model that kept a cache of the Phoenix Campaigns. We ended up removing it to eliminate complexity, although I've been daydreaming more and more about a Gambit Campaigns microservce to power the GET `/campaigns` and `/campaigns/:id` endpoints that the Gambit chatbot endpoint can query to get a rendered reply message for a given Campaign and template.